doc/usage/restructuredtext/directives.rst

```diff
@@ -373,8 +373,18 @@ units as well as normal text.
 
 .. rst:directive:: .. rubric:: title
 
-   This directive creates a paragraph heading that is not used to create a
-   table of contents node.
+   A rubric is like an informal heading that doesn't correspond to the document's structure,
+   i.e. it does not create a table of contents node.
+
+   .. rst:directive:option:: level: n
+      :type: number from 1 to 6
```

Comment:
I think ``level`` is a little vague, perhaps ``heading-level`` or ``section-level``?